zilliqa/src/consensus.rs

```diff
@@ -695,20 +695,6 @@ impl Consensus {
     /// Note that the algorithm below is mentioned in cfg.rs - if you change the way
     /// rewards are calculated, please change the comments in the configuration structure there.
     fn apply_rewards(
-        &mut self,
```

Comment:
Refactor here removing redundent fn. Removed `apply_rewards()` and renamed `apply_rewards_raw()` to `apply_rewards()`